cron_tasks.go

```diff
@@ -0,0 +1,27 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type CronTasksService interface {
+	CronTasksAdd(app string, params CronTasks) (*CronTasks, error)
```

Comment:
Why a pointer?

```suggestion
	CronTasksAdd(app string, params CronTasks) (CronTasks, error)
```

Reply:
Every others services work like this, I didn't really think about it

Reply:
And there is  `var _ AlertsService = (*Client)(nil)` in other services, I don't really know what this is for so I removed it, but maybe there's a purpose ?

Reply:
Yes there is a purpose. With this line, you can test if the structure `Client` implements the interface `AlertsService`. You may want to add after the interface declaration:

```
var _ CronTasksService = (*Client)(nil)
```